### PR TITLE
Remove explicit use of custom network

### PR DESCRIPTION
**Pull request by orenfromberg**
### Security Implications

- [ ] new software dependencies

- [ ] security controls or supporting software altered

- [ ] new data stored or transmitted

- [ ] security checklist is completed for this change

- [ ] requires more information or team discussion to evaluate security implications

- [ ] no PHI/PII is affected by this change

### CODE REVIEW

**Review by dhgreene — COMMENTED** [review]
We'll need some changes to `make down-v2` to match.
Screen Shot 2021-05-25 at 2 53 22 PM

**Comment by orenfromberg** [comment]
related to #1299

File: dpc-go/dpc-attribution/docker-compose.yml [hunk 1]
```diff
@@ -10,8 +10,6 @@ services:
       - POSTGRES_PASSWORD=dpc-safe
     ports:
       - "5434:5432"
-    networks:
-      - proxynet
```
Comment (orenfromberg) [inline dpc-go/dpc-attribution/docker-compose.yml:14]: is `proxynet` meant to be different from `my-proxy-net`?

Comment (jonfulk) [reply dpc-go/dpc-attribution/docker-compose.yml:14]: yeah... i was following an example I found online, but i think yours works and is cleaner. i'm checking it locally now

**Review by jonfulk — APPROVED** [review]

**Pull request merged**